xml/cha_images.xml

```diff
@@ -470,55 +470,101 @@
  <sect1 xml:id="sec-images-lifecycle">
   <title>Lifecycle of images</title>
   <para>
-   In the event of severe security vulnerabilities disclosed through the Common
-   Vulnerabilities and Exposures process (CVE), the &suse; public cloud
-   development team will refresh on-demand base images to ensure that new
-   instances launch in a patched state. This is in addition to the updates
-   available through the &suse; maintained update infrastructure.
+   All &suse; public cloud images follow a refresh cycle up to the point of
+   deletion. The refresh cycle follows a 'rolling' three month time frame. What
+   this means:
   </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Images are refreshed every three months. The new images replace the old
+     images in the exact state they were in. Replaced images are immediately
+     moving to the <literal>deleted</literal> state.
```

Comment:
s/deleted/deprecated/

Reply:
Thanks, this is wrong on https://www.suse.com/de-de/support/policy-products/#cloud, too.

Reply:
I'm on it